app/components/DosesDescriptionList/DosesDescriptionList.tsx

```diff
@@ -41,45 +43,23 @@ export const DosesDescription = ({
 export const DosesDescriptionList = ({
   dhbData,
 }: DosesDescriptionListProps) => (
-  <dl>
+  <dl className="flex flex-row">
     <DosesDescription
       term="First doses"
       hasMetTarget={dhbData.hasMetFirstDoseTarget}
       dosesPercent={dhbData.firstDosesPercentage}
     >
-      <dd className="flex-1 flex flex-row justify-content-end">
-        {dhbData.hasMetFirstDoseTarget ? (
-          <div className={styles["check"]}>
-            <CheckIcon aria-label="First doses target met" />
-          </div>
-        ) : (
-          <div className="data-text">
-            <strong>
-              {numberFormatter.format(dhbData.firstDosesTo90Percent)}
-            </strong>{" "}
-            left
-          </div>
-        )}
+      <dd className={cx(styles["vaccine-count"], "margin-top-px")}>
```

Comment:
Does this 1px do something functionally? Not desired from a design POV
```suggestion
      <dd className={cx(styles["vaccine-count"])}>
```

Reply:
Nothing functional, will remove!